Formula/rust.rb

```diff
@@ -86,7 +86,7 @@ def install
       ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
     end
 
-    args = %W[--prefix=#{prefix} --enable-vendor --set rust.jemalloc]
+    args = %W[--prefix=#{prefix} --sysconfdir=#{etc} --enable-vendor --set rust.jemalloc]
```

Comment:
This is a surprisingly old issue: https://github.com/rust-lang/cargo/issues/4070

I wonder why we didn't have this problem before.